client/my-sites/site-settings/related-posts/index.jsx

```diff
@@ -40,8 +40,14 @@ const RelatedPosts = ( {
 						text={ translate(
 							'Automatically displays similar content (related posts) at the end of each post.'
 						) }
-						link="https://jetpack.com/support/related-posts/"
+						link="https://en.support.wordpress.com/related-posts/"
```

Comment:
Do we want to hardcode the `en.` locale here? I think we could avoid that.

Reply:
Yup, `https://support.wordpress.com/related-posts/` seems to work 👍

Reply:
Switched link back to current.